stable_baselines3/common/base_class.py

```diff
@@ -472,11 +474,16 @@ def excluded_save_params(self) -> List[str]:
         """
         return ["policy", "device", "env", "eval_env", "replay_buffer", "rollout_buffer", "_vec_normalize_env"]
 
-    def save(self, path: str, exclude: Optional[List[str]] = None, include: Optional[List[str]] = None) -> None:
+    def save(
+        self,
+        path: Union[str, pathlib.Path, io.IOBase],
+        exclude: Optional[List[str]] = None,
+        include: Optional[List[str]] = None,
+    ) -> None:
         """
         Save all the attributes of the object and the model parameters in a zip-file.
```

Comment:
Update docs to reflect the new behaviour ("If directory does not exist, create one" etc). Same for other "save" functions.